Source/Core/EllipsoidGeodesic.js

```diff
@@ -173,6 +173,8 @@ define([
     }
 
     function computeProperties(ellipsoidGeodesic, start, end, ellipsoid) {
+        var scratchCart1 = new Cartesian3();
```

Comment:
Put these just above at file-level scope, not function-level.  These scratch variables are used to avoid allocating memory over and over again.